app/views/public/api_documentation.html.erb

```diff
@@ -523,8 +523,8 @@
 <pre>curl -H "Token: secret-api-token" 'https://www.validators.app/api/v1/sol-prices?from=2022-01-01T00:00:00'</pre>
 <p><strong class="text-muted">`to=%year-%month-%dayT%hour:%minute:%second`</strong> will return results until a given time, defaults to today. </p>
 <pre>curl -H "Token: secret-api-token" 'https://www.validators.app/api/v1/sol-prices?to=2022-01-01T00:00:00'</pre>
-<p><strong class="text-muted">`exchange=(ftx | coin_gecko)`</strong> will return results from a given exchange.</p>
-<pre>curl -H "Token: secret-api-token" 'https://www.validators.app/api/v1/sol-prices?exchange=ftx'</pre>
+<p><strong class="text-muted">`exchange=coin_gecko`</strong> will return results from a given exchange.</p>
```

Comment:
since we only have one exchange for now, exchange variable should not be needed

Reply:
Fixed.